{"name": "test_log3", "status": "failed", "statusDetails": {"message": "AssertionError: Error message not found on the page\nassert False", "trace": "test_nav = None\n\n    def test_log3(test_nav):\n        # Navigating to the homepage\n        driver.find_element(By.XPATH, \"//a[contains(text(),'home')]\").click()\n    \n        # Executing the test with incorrect credentials\n        driver.find_element(By.ID, \"email\").send_keys()\n        driver.find_element(By.ID, \"password\").send_keys()\n        driver.find_element(By.XPATH, \"//button[contains(text(),'Login')]\").click()\n        error_message_present = \"Please fill out the field\" in driver.page_source\n>       assert error_message_present, \"Error message not found on the page\"\nE       AssertionError: Error message not found on the page\nE       assert False\n\nTestcases\\test_Login.py:40: AssertionError"}, "start": 1706865697325, "stop": 1706865698976, "uuid": "1dca56fa-fa08-47ba-a52b-93abed056731", "historyId": "b124549d4c6ac177de77d5fe6176ab07", "testCaseId": "b124549d4c6ac177de77d5fe6176ab07", "fullName": "Testcases.test_Login#test_log3", "labels": [{"name": "parentSuite", "value": "Testcases"}, {"name": "suite", "value": "test_Login"}, {"name": "host", "value": "EDITH"}, {"name": "thread", "value": "2036-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Testcases.test_Login"}]}